### PR TITLE
Add new formula "Gomi v0.1.5"

### PR DESCRIPTION
**Pull request by b4b4r07**
[b4b4r07/gomi](https://github.com/b4b4r07/gomi)
> gomi is a simple trash tool that works on CLI, written in golang

### CODE REVIEW

File: Library/Formula/gomi.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        bin.install 'gomi'
+        system "curl -L https://raw.githubusercontent.com/b4b4r07/gomi/master/completions/zsh/_gomi >_gomi"
+        zsh_completion.install "_gomi"
+    end
+
+    test do
+        system "#{bin}/gomi", "--help"
+    end
+
+    def caveats; msg = <<-'EOS'
```
Comment (adamv) [inline Library/Formula/gomi.rb]: I don't think these caveats are needed.

File: Library/Formula/gomi.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+require "formula"
+
+class Gomi < Formula
+    homepage "https://github.com/b4b4r07/gomi"
```
Comment (adamv) [inline Library/Formula/gomi.rb]: Use two spaces for indents

File: Library/Formula/gomi.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    version "0.1.5"
+    url "https://github.com/b4b4r07/gomi/releases/download/v#{version}/gomi"
+    sha1 "2b3b5db55a6f6789fe8a649cc56f89f0fd487640"
+
+    def install
+        bin.install 'gomi'
+        system "curl -L https://raw.githubusercontent.com/b4b4r07/gomi/master/completions/zsh/_gomi >_gomi"
```
Comment (adamv) [inline Library/Formula/gomi.rb]: Use a `resource` to model this additional download.

Comment (b4b4r07) [inline Library/Formula/gomi.rb]: What is the "resourse"?

File: Library/Formula/gomi.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+require "formula"
```
Comment (mikemcquaid) [inline Library/Formula/gomi.rb]: This isn't needed.

File: Library/Formula/gomi.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+class Gomi < Formula
+  homepage "https://github.com/b4b4r07/gomi"
+  version "0.1.5"
+  url "https://github.com/b4b4r07/gomi/releases/download/v#{version}/gomi"
```
Comment (mikemcquaid) [inline Library/Formula/gomi.rb]: Hardcode the version in the `url`

File: Library/Formula/gomi.rb [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  sha1 "2b3b5db55a6f6789fe8a649cc56f89f0fd487640"
+
+  def install
+    bin.install 'gomi'
+    system "curl -L https://raw.githubusercontent.com/b4b4r07/gomi/master/completions/zsh/_gomi >_gomi"
```
Comment (mikemcquaid) [inline Library/Formula/gomi.rb]: This should be a `resource.

File: Library/Formula/gomi.rb [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    zsh_completion.install "_gomi"
+  end
+
+  test do
+    system "#{bin}/gomi", "--help"
```
Comment (mikemcquaid) [inline Library/Formula/gomi.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by mikemcquaid** [comment]
Self-submission of a two-week old project. Passing on this for now, sorry.

**Pull request closed without merging**